src/feedback/models/base.py

```diff
@@ -138,9 +138,11 @@ class FachgebietEmail(models.Model):
     email_sekretaerin = models.EmailField(blank=True)
 
     @staticmethod
-    def assign_to_fachgebiet(email):
+    def get_fachgebiet_from_email(email):
         try:
-            return Fachgebiet.objects.get(pk=FachgebietEmail.objects.get(email_suffix__contains=email).fachgebiet_id)
+            suffix = email.split('@')[1]
```

Comment:
use [-1] e.g. get the last element of the list to ensure that there is only an keyerror if the email is empty